src/ptychography40/reconstruction/ssb/udf.py

```diff
@@ -260,9 +261,9 @@ def get_task_data(self):
         ''
         result = super().get_task_data()
 
-        if self.meta.device_class == 'cpu':
+        if self.meta.array_backend in sparseconverter.CPU_BACKENDS:
```

Comment:
This makes me think - this code relies on the fact that LiberTEM is using `sparseconverter` under the hood.  This is an implementation detail which could change in the future. Probably LiberTEM should instead re-export these constants as part of its own API (somewhere under `libertem.common.sparse`?), meaning if we change sparse support in the future, we can update the re-exported constants, too.